test/sub_examples/fixtures/raw_pymodbus.log

```diff
@@ -0,0 +1,14 @@
+2023-07-07 15:14:33,158 - DEBUG - pymodbus.logging - Running transaction 28107
```

Comment:
This file should not be needed.

Reply:
It's used in testing, unless we on-the-fly generate a log file.  Probably better to do that

Reply:
Removed this file with tests entirely per [this request](https://github.com/pymodbus-dev/pymodbus/pull/1665#discussion_r1267615225)